atd-vze/src/views/Crashes/RelatedRecordsTable.js

```diff
@@ -200,31 +201,22 @@ const RelatedRecordsTable = ({
                                     }
                                     defaultValue={
                                       // Check for null values and display as blank
-                                      row[field] &&
-                                      row[field][`${fieldLookupPrefix}_id`] !==
-                                        null
-                                        ? row[field][`${fieldLookupPrefix}_id`]
+                                      row[field] && row[field][`id`] !== null
+                                        ? row[field][`id`]
                                         : ""
                                     }
                                     type="select"
                                   >
-                                    {/* Show a NO DATA option only when formatValue is displayed. */}
-                                    {formatValue(row, field) === "NO DATA" && (
-                                      <option value={null}>NO DATA</option>
-                                    )}
+                                    <option value={""}>NO DATA</option>
```

Comment:
it's pretty weird to have this option now, because selecting "NO DATA" resets the field to the CRIS-provided value. i'm going to make a separate issue for us to address this holistically, and we can leave this as-is for now?

Reply:
yeah also fwiw this isnt a change that was made in this PR, i guess the data collector branch didnt have these updates from my [recent PR](https://github.com/cityofaustin/atd-vz-data/pull/1435)?